### PR TITLE
Squash migrations #167

### PR DESCRIPTION
**Pull request by glebm**
1. Squashes the migrations into one database-agnostic migration.
2. Removes schema files.
3. Adds sqlite3 support and defaults to it, resolving #142.
4. Re-creates the test database and runs the migrations before rspec.

Σ: Simplifies development massively!

---

Before merging this, please tag 0.2.0, so there is a clear cut-off point for the migrations change.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -102,41 +104,6 @@ end
...
-  end
-
-  desc 'Truncate all tables'
-  task truncate: :environment do
-    ActiveRecord::Base.connection.tables.each do |table|
-      ActiveRecord::Base.connection.execute("TRUNCATE TABLE #{table} CASCADE;")
-    end
-  end
-end
-
```
Comment (jayroh) [inline Rakefile]: Huuuuuuuge

**Comment by jayroh** [comment]
I have zero comments outside of 



Will let you know as soon as we get 0.2.0 tagged and released.

**Comment by glebm** [comment]
Let's tag v0.2.1 as soon as this is merged, so we don't have too many users with lots of tiny migration files.

**Comment by jayroh** [comment]
Gem v0.2.0 and v0.2.1 both pushed to rubygems  :)

**Pull request merged**